### PR TITLE
replace uservoice with docs.metamaps.cc + Hylo

### LINKED ISSUE #754: Replace uservoice and update help box
Before Oct 21, I'd like to

1. Establish our new community, probably on Hylo
2. Replace Uservoice in the code with similar functions that link to the Hylo community
3. Update the README here to point to the Hylo community
4. Update the help box based on Ben's recommendations

After that (but not before closing this issue), we can migrate and update articles from Uservoice to Hylo.

I can take point on coding this and writing some articles, but not all articles 😅

### PR DESCRIPTION
**Pull request by devvmh**
Closes #754 

One downside with this PR is that it removes the little 'Feedback' popup on the side. We do have a feedback map on metamaps.cc; it would be cool to link to it somehow

TODO
- [ ] verify that the video tutorials are still accurate (enough) for rock 'n' roll

### CODE REVIEW

File: app/assets/stylesheets/uservoice.css.erb [hunk 1]
```diff
@@ -1,25 +0,0 @@
-/* =USERVOICE ICON DEFINE
---------------------------------------------------------*/
-
-.unauthenticated .uv-icon {
-	display: none;
-}
-
-div.uv-icon.uv-bottom-left {
-	background-image: url(<%= asset_data_uri 'feedback_sprite.png' %>);
```
Comment (Connoropolous) [inline app/assets/stylesheets/uservoice.css.erb]: we could leave this stuff in and make it a link... change the classnames or not bother

**Comment by Connoropolous** [comment]
other than my one thought, this looks great!

File: app/assets/stylesheets/uservoice.css.erb (hunk 1 above)
Comment (devvmh) [inline app/assets/stylesheets/uservoice.css.erb]: what would it link to? the feedback map? I'm interested in doing this

**Comment by Connoropolous** [comment]
I made the change (feedback thing as link to hylo)

**Pull request merged**